src/api/java/openiss/Kinect.java

```diff
@@ -29,11 +29,17 @@
 package openiss;
 
 import java.awt.image.BufferedImage;
+import java.io.File;
 import java.lang.reflect.Method;
-import java.nio.ByteBuffer;
-import java.nio.ShortBuffer;
-import java.nio.FloatBuffer;
-
+import java.nio.*;
```

Comment:
It is proper to expand `java.nio.*` to what modules are actually used instead of bulk import of everything.